en/editor/publish/native-options.md

```diff
@@ -1,195 +1,218 @@
 
 #### Target API Level
 
-Set up the Target API Level required for compiling the Android platform. Click the **Set Android SDK** button next to it to quickly jump to the configuration page. Refer to the [Setup Native Development Environment](setup-native-development.md) documentation for specific configuration rules.
+Set the Target API Level required when compiling the Android platform. Click the **Set Android SDK** button next to it to quickly jump to the configuration page. For specific configuration rules, please refer to the [Configure Native Release Environment Path](setup-native-development.md#%E9%85%8D%E7%BD%AE%E5%8E%9F%E7%94%9F%E5%8F%91%E5%B8%83%E7%8E%AF%E5%A2%83%E8%B7%AF%E5%BE%84) documentation.
 
 #### APP ABI
 
-Set up the CPU types that Android needs to support, including **armeabi-v7a**、**arm64-v8a** and **x86**. You can choose one or more options.
+Set the CPU type that Android needs to support. It is possible to choose one or more options. Currently, there are three types: **armeabi-v7a**, **arm64-v8a**, and **x86**.
 
-> **Notes**:
-> 1. When you select an ABI to build and then build another ABI without `Clean`, both ABI's `so` will be packaged into the APK, which is the default behavior of Android Studio. If you import a project with Android Studio, after selecting an ABI to build, run **Build -> Clean Project**, then build another ABI, only the latter ABI will be packaged into the APK.
-> 2. After the project is imported with Android Studio, it is an independent existence and does not depend on the **Build** panel. If you need to modify the ABI, you can directly modify the `PROP_APP_ABI` property in `gradle.properties` file as shown below:
+> **Note**:
 >
->     ![modify abi](publish-native/modify_abi.png)
-
-#### Use Debug Keystore
+> 1. After you select an ABI to build, build another ABI without cleaning. At this time, the so of both ABIs will be packaged into the apk. This is the default behavior of Android Studio. If you use Android Studio to import the project, after selecting an ABI to build, first execute **Build -> Clean Project** and then build another ABI. At this time, only the latter ABI will be packaged into the apk.
+>
+> 2. After the project is imported with Android Studio, it is an independent existence and does not depend on the build release panel. If you need to modify the ABI, just modify the **PROP_APP_ABI** property in the **gradle.properties** file.
+>
+> ![modify abi](publish-native/modify_abi.png)
 
-Android requires that all APKs be digitally signed with a certificate before they can be installed. A default keystore is provided, check the **Use Debug Keystore** to use the `default keystore`. If you need to customize the keystore, you can remove the **Use Debug Keystore** checkbox. Please refer to the official [Android Documentation](https://developer.android.com/studio/publish/app-signing) for details.
+#### Use the debug keystore
```

Comment:
Except for the case usage, when should we keep "the" and when should we delete it in a title?

Reply:
you keep it, almost always. I read out loud to test it.

Reply:
OK.